openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2290,6 +2282,53 @@ The following is a non-normative example of a Credential Response containing a C
 
 <{{examples/credential_response_sd_jwt_vc.txt}}
 
+# Wallet Attestations {#walletattestation}
+
+The Wallet attestation defined by this specification is an interoperable, verifiable statement that provides evidence of authenticity and security properties of a Wallet to the Authorization Server. Wallets may have different architectures, e.g. they may be app-based with a backend service or web Wallets running entirely within the browser. Wallet attestations are independent from the Wallet's architecture, but require a require a backend service from the Wallet Provider. When the Wallet Provider creates the wallet attestation, it needs to verify the authenticity of the client. App attestation, as provided by mobile operating systems, e.g., iOS's DeviceCheck or Android's Play Integrity, allows the Wallet Provider's backend system to ensure it is communicating to a legitimate instance of its genuine app. Those mechanisms can be utilized to validate the internal integrity of the Wallet.
```

Comment:
In my opinion the Wallet Attestation should be a standardised credential that can (and should) be displayed to the user just like his/her other credentials. This will help to engender user trust in the entire ecosystem, and in the wallet app in particular. We should define the standard format for this wallet attestation credential.

Reply:
applied with minor changes.

Reply:
@David-Chadwick this PR defines a standardised credential for wallet attestation. It should be displayed to the user, but not as any other credential, as that may confuse most users, please re-raise this point if its important to you

Reply:
The user may hold credentials for multiple subjects e.g. themselves, their pet, their children, their wallet. So I do not see why the wallet attestation cannot be displayed like any other credential. It just needs the wallet to have tabs labelled with the subject's name/ID/alias, so that the user can see which credentials they hold for which subjects.